hw2/05-form.html

```diff
@@ -1,20 +1,128 @@
+                            >Registration Status:</label
+                        ><br />
+                        <select
+                            class="w-100 rounded-2"
+                            name="regStatus"
+                            id="regStatus"
+                            required
+                        >
+                            <option value="" selected disabled>
+                                Choose an Option
+                            </option>
+                            <option value="Registered">Registered</option>
+                            <option value="Not registered">
+                                Not Registered
+                            </option>
+                        </select>
+                    </div>
+                    <fieldset>
+                        <legend class="mt-3 fw-bold fs-6">
+                            Have you taken any of the following courses?
+                        </legend>
+                        <br />
+                        <label for="course1">
+                            <input
+                                type="checkbox"
+                                name="coursesTaken"
+                                id="course1"
+                                value="Programming Languages"
+                            />
+                            <span>Programming Languages</span> </label
+                        ><br />
+                        <label for="course2">
+                            <input
+                                type="checkbox"
+                                name="coursesTaken"
+                                id="course2"
+                                value="Operating Systems"
+                            />
+                            <span>Operating Systems</span> </label
+                        ><br />
+                        <label for="course3">
+                            <input
+                                type="checkbox"
+                                name="coursesTaken"
+                                id="course3"
+                                value="Full Stack Web Development"
+                            />
+                            <span>Full Stack Web Development</span>
+                        </label>
+                        <br />
+                    </fieldset>
+                    <div>
+                        <label for="comment" class="mt-2">Anything else ?</label
+                        ><br />
+                        <textarea
+                            class="w-100 rounded-2"
+                            name="comment"
+                            id="comment"
+                            cols="40"
+                            rows="3"
+                        ></textarea>
+                    </div>
+                    <div class="d-flex p-2">
+                        <button
+                            class="bg-primary text-white m-1 w-50 rounded-2"
```

Comment:
Style - Button spacing and styling bit off. Minor.

Reply:
removed the p-2 padding.  spacing is a little better now.